docs/Secure-Coding-Guide-for-Python/CWE-703/CWE-230/compliant02.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: OpenSSF project contributors
+# SPDX-License-Identifier: MIT
+""" Compliant Code Example """
+
+import math
+
+
+def balance_is_positive(value: str) -> bool:
+    """Returns True if there is still enough value for a transaction"""
+    _value = float(value)
+    if math.isnan(_value) or _value is None:
+        raise ValueError("Expected a float")
+    if _value <= 0:

```

Comment:
This might be slightly confusing because `print(balance_is_positive("0.001"))` returns false in compliant01.py but returns true here. If we assume the balance uses cents, we could just change the threshold in this condition:
```suggestion
    if _value < 0.01:
```
If we do so, the same should be done in noncomplaint01.py so that this part of the code remains unchanged.